docs/content/preview/architecture/docdb/persistence.md

```diff
@@ -130,10 +130,13 @@ Testing the packed row feature with different configurations showed significant
 
 ### Limitations
 
-The packed row feature works for the YSQL API using the YSQL-specific GFlags with most cross features like backup and restore, schema changes, and so on, subject to certain known limitations which are currently under development:
+The packed row feature for the YSQL API works across all cross features like backup and restore, schema changes, and so on, subject to certain known limitations which are currently under development:
 
-* [#15143](https://github.com/yugabyte/yugabyte-db/issues/15143) Colocated and xCluster - There are some limitations around propagation of schema changes for colocated tables in xCluster in the packed row format that are being worked on.
-* [#14369](https://github.com/yugabyte/yugabyte-db/issues/14369) Packed row support for YCQL is limited and is still being hardened.
+* [#20638](https://github.com/yugabyte/yugabyte-db/issues/20638) Colocated and Packed Row - There is an issue with aggressive garbage collection of the schema versions that are stored in DocDB, in order to interpret Packed Row data. This issue is limited to the colocated table setting, and manifests in certain flavors of compactions. Since it results in non-recoverable errors for colocated workloads, the gflag `ysql_enable_packed_row_for_colocated_table` can be set to false, to avoid the issue on 2.20.1 builds, while the team works to address the issue.
```

Comment:
"avoid the issue on 2.20.1 builds" - unclear whether this means the flag is available only in 2.20.1 (or 2.20.1 and later?), or if this is the only way to avoid the issue in 2.20.1.